### DISCUSSION TITLE
November Challenge

### DISCUSSION THREAD

**Discussion by aurelieverrot** [Virtual-Coffee/VC-Community-Docs#213] [Monthly Challenges]
November is National Novel Writing Month.
Like last year, VC will participate and we will try to collectively reach 50,000 words with our blog posts.

It can be a tech article or tech-related (career, feedback, ...).

Those who participated last year, is there anything we need to know, add, support for this challenge?
Do you think it could be a good idea to give 4 themes (one per week) and make our members use them to facilitate the writing process?

I would like to innovate a little bit and have an *opt-in* challenge on top of it: win a pack of stickers!
Conditions would be to reach 5,000 words to be selected for a raffle. The member who gets picked would need to share their home address with me.
Worst case scenario it will cost me an international stamp, really nothing.
What do you think?

**Comment by dominicduffin1** [comment]
I have an idea for National Novel Writing Month, this is either an amazing idea or a terrible one, but we could write a piece of fiction loosely set in the tech industry with each participant contributing one short chapter. What do people think?

**Comment by BekahHW** [comment]
This is the challenge from last year: https://virtualcoffee.io/monthlychallenges/nov-2020/.

@dominicduffin1 we could definitely do that in addition to the blog writing. I think it would be good to gauge community interest first. Now might be a good time to do that. I wonder if it's better to ask in slack or on a VCIO discussion board, just so we don't lose it. 

@aurelieverrot I think having a list of writing options would be a good idea, but not making them requirements. Decreasing barriers to entry is one of my biggest goals for most of the challenges. People seemed to really enjoy the team goal of this challenge, so I think we should keep that part going. Although--at some pt--ppl were talking about who was the individual leader.

**Comment by ClJarvis** [comment]
@aurelieverrot I pinned these in our content-creation channel but will add here for the discussion. One is a list of 100 blog topics. The other is a seven day course on improving your developer blog. It's a good resource for  bloggers. 

https://dev.to/andrewbaisden/100-blog-topic-ideas-for-your-next-article-no-more-writers-block-2e0j. 

https://bloggingfordevs.com/

**Comment by ClJarvis** [comment]
I shared this in the VC idea channel and it sort ties in here.  For December,. What if we did a Virtual Coffee advent of content. With blog posts from different members? a guess blogger a day thing. 
Have members submit their favorite post. Mention it when the Nov challenge is announced. We wouldn't have to limit it to Nov either, any blog post form the year. It could also function as a VC member review of the year.

For the Nov 2020 we had 18 different bloggers, We've grown since then so may have enough content.

**Comment by davidalpert** [comment]
oh my, this may have to be the month I formally ramp up my writing habit another notch. it's going to be fun!
